expressions/methods.md

```diff
@@ -228,6 +228,47 @@ primitive Foo
     f.>add_option(o1).>add_option(o2).make_object() // Works. The expression returns an Object
 ```
 
+## Behaviours
+Behaviours are asynchronous method calls. As such, they cannot return anything, and the caller will not wait for the call to finish. Here is a quick example to illustrate this certainly :
```

Comment:
> Behaviours are asynchronous method calls.

To be precise (i think we need to be here to avoid confusion), behaviours are not method calls, but methods that are asynchronously evaluated in the context of their containing actor, that is possibly on another thread.

Maybe it is worth mentioning that they also cannot error.